diff --git a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
index 59ea66e..a6d09ed 100644
--- a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
+++ b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
@@ -1757,7 +1757,7 @@ function parseDisplay(display, options) {
     }
 }
 function parseAspectRatio(aspectRatio) {
-    if (aspectRatio.auto) {
+    if (aspectRatio.auto || !aspectRatio.ratio) {
         return "auto";
     }
     else {
diff --git a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
index bf095a5..a91a3b1 100644
--- a/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
+++ b/node_modules/react-native-css-interop/src/css-to-rn/parseDeclaration.ts
@@ -2639,7 +2639,7 @@ function parseAspectRatio(
   // This is missing types
   aspectRatio: any,
 ): RuntimeValueDescriptor {
-  if (aspectRatio.auto) {
+  if (aspectRatio.auto || !aspectRatio.ratio) {
     return "auto";
   } else {
     if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
